base/linalg/lapack.jl

```diff
@@ -3759,7 +3759,7 @@ for (stev, stebz, stegr, stein, elty) in
             chklapackerror(info[])
             if any(ifail .!= 0)
                 # TODO: better error message / type
-                error("failed to converge eigenvectors:\n$(nonzeros(ifail))")
+                error("failed to converge eigenvectors:\n$(find(x->x != 0, ifail))")
```

Comment:
There is no `nonzeros` method for `Vector` on master, but with this excision we get
```
WARNING: importing deprecated binding Base.nonzeros into LAPACK.
```
here, so updated.

Reply:
Perhaps `!iszero`? :)